Framework/PythonInterface/plugins/algorithms/LoadEXED.py

```diff
@@ -38,16 +47,29 @@ def PyInit(self):
                                           extensions = ["xml"]),
                              doc="Instrument definition file. If no file is specified, the default idf is used.")
 
+        self.declareProperty('AngleOverride', -255.0,
```

Comment:
Can this property (not added at the end) break some existing user scripts?

Reply:
This algorithm was prepared by the instrument scientist. He is fine with that..

Reply:
Ok, then it should not be a problem.